# List of all AprilKinect detections from a single camera frame.
Header header
AprilKinectDetection[] detections
